i18n/MonotoneMuseum/ko_kr.yml

```
@@ -25,65 +25,538 @@ ko_kr:
+        - |-
+          【운명의 총애】
+          총애 혹은 비극의 대가. 당신은 이탈능력을 이 공연에 한해 추가로 하나 더 사용할 수 있게 된다(임의로 선택할 것). 선택한 이탈능력의 박탈치 상승은 +1된다.
+      DTO:
+        name: 일그러짐표(야외)
+        items:
+        - |-
+          【죽음만연】
+          나무는 마르고 기괴한 모습이 되며 꽃은 시든다. 대기는 탁한 썩은내를 뿜으며 바위는 뒤틀린다. 모든 것이 죽음의 기운에 감싸인다.
+        - |-
+          【왜곡표】
+          일그러진 이정표가 내려온다. 일그러짐표（『MM』P263）를 굴린다.
+        - |-
+          【계절변용】
+          계절이 갑자기 변화한다. 1D6을 굴려 1이면 봄, 2면 여름, 3이면 가을, 4는 겨울, 5라면 사계절이 미친듯이 변동되고 6이면 GM이 임의로 정한다. 급격한 계절의 급격한 변화가 자연과 사람들의 건강을 해친다.
+        - |-
+          【올풀림】
+          세계에 금이 가고 올풀림이 나타난다. 올풀림에 닿은 자는 허무에 삼켜져 돌아올 수 없다.
+        - |-
+          【이형화】
+          주변의 동물들이 일그러짐때문에 이형으로 변해 습격해온다. 이형 무리（『IZ』P237）×1D6체와 전투를 해도 좋다.
+        - |-
+          【동결세계】
+          기온이 비정상적으로 저하해 모든 것이 얼어붙는다. 또한 얼어붙은 세계에서 전투하는 모든 캐릭터는 방어판정에서 펌블이 날 경우 낭패를 받는다.
+        - |-
+          【기상폭주】
+          태풍, 용오름, 눈보라, 폭염 등 이상기후가 발생한다. 배는 나뭇잎처럼 흔들리고 짐승은 공포에 허둥댄다. 이 씬 내에서 진행하는 모든 이동의 이동 거리가 반감한다.
+        - |-
+          【하늘의 분노】
+          일그러짐에 의해서 있을 수 없는 이상기후가 발생한다. 모든것을 날려보내는 바람이 불고 뇌명이 운다. 이 씬 중 모든 캐릭터는 비행상태가 될 수 없다.
+        - |-
+          【미궁의 숲】
+          주변의 풍경이나 지형이 지식이나 지도와 전혀 맞지 않는다. 어딘가로 향할 때 【지각】 난이도 12의 판정에 실패한 경우 헤멘 결과 【MP】 2D6점을 잃는다.
+        - |-
+          【죽음의 세계】
+          세계는 기묘하도록 아름답다. 물에는 물고기 한 마리 없고 덤불에는 벌레 한 마리 없다. 하늘에는 새가 날지 않는다. 기분나쁜 정적이 세계를 감싼다.	
+        - |-
+          【그림자극】
+          주변 어드메의 그림자에서 실잣는이들의 (혹은 등장한 NPC의) 불길한 운명을 암시하는 오랫소리가 들려온다. 그것은 이형의 소리나 허무가 삐걱인느 소리 혹은 일그러짐의 환각일지도 모른다.
+        - |-
+          【색채침식】
+          세계가 칠흑과 둔백으로 덧칠된다. 씬 내에 존재하는 여러 무생물과 생물이 흑백의 모노톤이 된다. 실잣는이는 【재봉】 난이도 8의 판정에 성공하면 이 영향을 받지 않아도 좋다.
+        - |-
+          【농무】
+          설령 그곳이 땅속이라도 둔백의 농무가 주변을 휘감는다. 손을 뻗으면 그 손끝이 보이지 않을 정도다. 농무 속에서의 공격은 적과의 거리를 2배로 계산한다.
+        - |-
+          【이형출현】
+          올풀림에서 사람을 습격하는 것만을 생각하는 이형이 나타나 공포에 떠는 사람을 공격한다. GM이 원한다면 이형의 무리 (IZ 237p)x1D6체와 전투를 해도 좋다. 또한 GM은 상황에 따라 다른 에너미를 선택해도 좋다.
+        - |-
+          【광맥출현】
+          있을수 없을 정도로 풍부한 광맥이 나타난다. 황금, 보석, 바다라면 산호 등이다. 사람들이 안색을 바꾸며 광맥으로 몰려든다. 그로 인해 발생하는 혼란은 GM이 결정할 것.
+        - |-
+          【허무출현】
+          올풀림 속에서 허무가 새어나와 약한 이들을 끌고들어간다. 씬에 등장하고 있는 엑스트라는 벌레 한 마리까지 소멸하고 두 번 다시 나타나지 않는다.
+        - |-
+          【오염세계】
+          물은 독을 품고, 대기는 유독성 안개로 가득하다. 등장하고 있는 모든 캐릭터는 【체력】난이도 10의 판정을 해서 실패할 경우 [중독] 5를 받는다. 엑스트라가 어떻게 될지는 GM이 정하지만 치료하지 않는 한 얼마 안 가 사망할 것이다.
+        - |-
+          【지역소멸】
+          연극의 무대가 된 지역 그 자차체가 허무에 삼켜져 사라진다. 영향하의 모든 캐릭터(가람을 포함)는 【재봉】 난이도 10의 판정에 성공하면 탈출할 수 있다. 실패한 경우 즉시 사망한다. 엑스트라는 무조건 사망한다.
+      DTS:
+        name: 일그러짐표(바다)
+        items:
+        - |-
+          【해양오염】
+          적조가 발생한다. 혹은 해수 그 자체가 독으로 변한다. 등장한 모든 캐릭터는 【체력】 난이도 10의 판정을 해 실패할 경우 [중독] 5를 받는다. 엑스트라가 어떻게 될지는 GM이 정하지만 치료하지 않는 한 얼마 안 가 사망할 것이다.
+        - |-
+          【색채침식】
+          세계가 칠흑과 둔백으로 덧칠된다. 씬 내에 존재하는 여러 무생물과 생물이 흑백의 모노톤이 된다. 실잣는이는 【재봉】 난이도 8의 판정에 성공하면 이 영향을 받지 않아도 좋다. 
```

The trailing space should be removed.
```suggestion
          세계가 칠흑과 둔백으로 덧칠된다. 씬 내에 존재하는 여러 무생물과 생물이 흑백의 모노톤이 된다. 실잣는이는 【재봉】 난이도 8의 판정에 성공하면 이 영향을 받지 않아도 좋다.
```